CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
-# Changelog
-0.9.0 (2018-05-14)
-==================
+### [Next Release Here]
 
-First beta release of libledger-core for Ledger Vault integration
\ No newline at end of file
+> 2019/??/??
+
+- Change the internal representation of `DynamicObject`. The new representation is optimized to
+  take less memory and to be safer in terms of typing. The internal interfaces were also
+  redesigned to allow for a smoother transition to C++17’s `std::variant` when we see fit.
+
+## 2.5.0
+
+> 2019/01/22
+
+- Resetting the libcore to its fresh state is now possible via the `WalletPool::freshResetAll`
+  function.
+
+## 2.4.0
+
+> 2019/01/22
+
+- Add the possibility to switch off the internal logger of libcore — this rely on the
+  `DynamicObject` that now has a new boolean configuration, `"ENABLE_INTERNAL_LOGGING"`. Set it
+  to `true` to enable the internal logging (default), otherwise, set to `false` to completely
+  disable internal logging (this also will prevent the core library from creating log files).
+
+## 2.3.0
+
+> 2019/01/21
+
+- Change the way the library is compiled and linked by using explicti interface libraries. This
+  also fixes a Windows issues related to non-explicitly exported symbols.
+
+## 2.2.0
+
+> 2019/01/15
+
+- Add a new mechanism to `DatabaseEngine`. This allows for custom database driver, which should
+  allow database switching in easier ways.
```

Comment:
IMHO it's better to say : 'This allows custom database driver, which allow switching between database backends easier.'

Reply:
Yep!